packages/embed/src/popup/popup.ts

```diff
@@ -22,14 +24,16 @@ export const createPopupController = (
   })
 
   subject.approvalLost$.subscribe(() => {
-    popup.current.stopCallback()
-    popup.current.popup.close()
-    subject.approvalStarted$.next()
+    if (popup.current) {
```

Comment:
`approvalLost$` should not fire from the overlay as their is no button to relaunch the popup?
This change makes sense though.

Reply:
In the case of GPay, no, there is no reopen button. 👍